tests/wpunit/Tribe/Tickets/Commerce/PayPal/Handler/IPNTest.php

```diff
@@ -27,54 +27,70 @@ private function make_instance() {
 	 * Test get_config_status with empty option
 	 */
 	public function test_get_config_status_with_empty_option() {
-		$this->markTestSkipped( 'This came from UNIT tests we need to remove define_function usage and make sure it works.' );
-//		define_function( 'tribe_get_option', function () {
-//			return '';
-//		} );
+		// Mock tribe_get_option to return empty values using the global filter
+		add_filter( 'tribe_get_option', function( $value, $option_name, $default ) {
+			$map = [
+				'ticket-paypal-email'           => '',
+				'ticket-paypal-ipn-enabled'     => '',
+				'ticket-paypal-ipn-address-set' => '',
+			];
+
+			if ( isset( $map[ $option_name ] ) ) {
+				return $map[ $option_name ];
+			}
+
+			return $value;
+		}, 10, 3 );
 
 		$ipn = $this->make_instance();
 
 		$this->assertEquals( 'incomplete', $ipn->get_config_status() );
 		$this->assertEquals( 'incomplete', $ipn->get_config_status( 'slug' ) );
-		$this->assertInternalType( 'string', $ipn->get_config_status( 'label' ) );
+		$this->assertIsString( $ipn->get_config_status( 'label' ) );
 		$this->assertEquals( 'complete', $ipn->get_config_status( 'slug', 'complete' ) );
-		$this->assertInternalType( 'string', $ipn->get_config_status( 'label', 'complete' ) );
+		$this->assertIsString( $ipn->get_config_status( 'label', 'complete' ) );
 		$this->assertEquals( 'incomplete', $ipn->get_config_status( 'slug', 'incomplete' ) );
-		$this->assertInternalType( 'string', $ipn->get_config_status( 'label', 'incomplete' ) );
+		$this->assertIsString( $ipn->get_config_status( 'label', 'incomplete' ) );
 		$this->assertFalse( $ipn->get_config_status( 'slug', 'foo' ) );
 		$this->assertFalse( $ipn->get_config_status( 'label', 'foo' ) );
 
+		// Clean up filter
+		remove_all_filters( 'tribe_get_option' );
 	}
 
 	/**
 	 * Test get_config_status with complete option
 	 */
 	public function test_get_config_status_with_complete_option() {
-		$this->markTestSkipped( 'This came from UNIT tests we need to remove define_function usage and make sure it works.' );
-//		define_function( 'tribe_get_option', function ( $name ) {
-//			$map = [
-//				'ticket-paypal-email'           => 'foo@bar.baz',
-//				'ticket-paypal-ipn-enabled'     => 'yes',
-//				'ticket-paypal-ipn-address-set' => 'yes',
-//			];
-//			if ( ! isset( $map[ $name ] ) ) {
-//				throw new \RuntimeException( "No call was expected for option {$name}" );
-//			}
-//
-//			return $map[ $name ];
-//		} );
+		// Mock tribe_get_option to return complete configuration values using the global filter
+		add_filter( 'tribe_get_option', function( $value, $option_name, $default ) {
+			$map = [
+				'ticket-paypal-email'           => 'foo@bar.baz',
+				'ticket-paypal-ipn-enabled'     => 'yes',
+				'ticket-paypal-ipn-address-set' => 'yes',
+			];
+
+			if ( isset( $map[ $option_name ] ) ) {
+				return $map[ $option_name ];
+			}
+
+			return $value;
+		}, 10, 3 );
```

Comment:
If e do it this way, we only hit these specific keys instead of running for every `tribe_get_option` call:

```suggestion
		add_filter( "tribe_get_option_ticket-paypal-email", function( $value ) {
			return 'foo@bar.baz';
		}, 10, 1 );

		add_filter( "tribe_get_option_ticket-paypal-ipn-enabled", function( $value ) {
			return 'yes';
		}, 10, 1 );

		add_filter( "tribe_get_option_ticket-paypal-ipn-address-set", function( $value ) {
			return 'yes';
		}, 10, 1 );
```

Reply:
Same for the others 😉

Reply:
Oh, and the param count is probably moot there - since the default is 1.